code/style.css

```diff
@@ -2,6 +2,309 @@
 in this case it is the div with the answer class next to the div with the active class.
 this can be used for the FAQ-accordion. read more here: https://www.w3schools.com/css/css_combinators.asp */
 
+
+body{
+  background-color: rgb(210, 238, 215);
+  margin: 0px;
+  font-family: Verdana, Geneva, Tahoma, sans-serif;
+}
+
+/* ----------hero-header------------ */
+.hero{
+  width: 100%;
+  height: 660px;
+  background-image: url("./images/hero-image.jpg");
+  background-repeat: no-repeat;
+  background-position: center;
+  background-size: cover;
+  object-fit: contain;
+  position: relative;
+  display: flex;
+  justify-content: space-between;
+}
+
+.hero >h2 {
+  color:magenta;
+  font-size: 1rem;
+  margin-left: 10px;
+  font-style: italic;
+}
+
+.hero-menu{
+  display: flex;
+  justify-content: space-evenly;
+  display: none;
+  color: rgb(235, 234, 220);
+  font-size: 1rem;
+  gap: 20px;
+  width: 100%;
+}
+
+.hero-text {
+  text-align: center;
+  position: absolute;
+  top: 80%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  width: 100%;
+}
+
+.hero-text > h3{
+  color: rgb(31, 44, 233);
+}
+
+.hero-text > button {
+  background-color: rgb(8, 116, 57);
+  color: white;
+  border-radius: 5%;
+  padding: 10px;
+  margin: 10px;
+}
+
+.hero-text > button:hover {
+ transform: scale(1.05);
+}
+
+/* animation to change color of text header */
+.hero-text >h1 {
+  margin: 10px 0px;
+  animation: heroText 5s infinite alternate-reverse;
+  width: 100%;
+}
+
+@keyframes heroText {
+   0%  {color: rgb(49, 84, 243);}
+  25%  {color: yellow;}
+  50%  {color: rgb(51, 255, 0);}
+  100% {color: rgb(41, 209, 195);}
+}
+
+.form_accordion{
+  display: flex;
+  flex-direction: column;
+}
+
+/* ------------form--------------- */
+.form {
+  font-size: 1rem;
+  border: 1px solid gray;
+  background-color:rgb(145, 167, 240);
+  margin: 20px 10px;
+  text-align: center;
+  padding: 1rem;
+  border-radius: 5%;
+  color: aliceblue;
+  max-width: 100%;
+}
+
+.form h2{
+  margin: 20px 0px;
+}
+
+.form-item{
+  display: flex;
+  flex-direction: column;
+}
+
+.text-type{
+  align-self: center; 
+}
+
+.text-type input{
+  min-width:190px;
+  min-height: 17px;
+  max-width: 200px;
+  max-height: 19px;
+  margin: 5px 0px;
+  display: block;
+}
+
+.text-type label{
+  width: 100%;
+  display: block;
+}
+
+
+.radio-type{
+  display: flex;
+  justify-content: center;
+  padding: 15px;
+
+}
+
+.check-type{
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 15px;
+  gap: 5px;
+}
+
+.submit-type{
+  width: 100%;
+  display: flex;
+  justify-content: center;
+  margin: 10px 0px;
+}
+
+.submit-type button{
+  border-radius: 10%;
+  background-color: green;
+  color: honeydew;
+  padding: 10px;
+}
+
+label{
+  font-size: 0.8rem;
+}
+
+input:focus{
+  background:mistyrose
+}
+
+input [type="text"]{
+  border: 1px solid rgb(109, 108, 108);
+}
+
+input[type="radio"]{
+ filter: hue-rotate(45deg);
+}
+
+input[type="checkbox"]{
```

Comment:
Clever way of changing the color of the checkboxes/radio buttons without too much effort.